recipe/conda_build_config.yaml

```diff
@@ -531,7 +531,8 @@ pango:
 perl:
   - 5.26
 pixman:
-  - 0.34
+  - 0.34  # [not (aarch64 or ppc64le)]
+  - 0.38  # [aarch64 or ppc64le]
```

Comment:
Let's migrate everything to 0.38 then.

Reply:
That would require a migrator right?

Reply:
Yep. Those are relatively easy and won't affect your plans to re-render what you need for `aarch64` or `ppc64le` right way.

Reply:
ok done.